bodhi/server/scripts/approve_testing.py

```diff
@@ -107,6 +109,11 @@ def main(argv=sys.argv):
 
                 notifications.publish(update_schemas.UpdateRequirementsMetStableV1.from_dict(
                     dict(update=update)))
+
+                if update.autotime and update.days_in_testing >= update.stable_days:
+                    print(f"Automatically marking {update.alias} as stable")
+                    update.set_request(db=db, action=UpdateRequest.stable, username="bodhi")
```

Comment:
If the user sets ```autokarma``` off and the update reaches the karma threshold, this code will not execute because the condition above on line 91 will be ```True```.

Please add a test case for that situation so we can ensure that the right thing happens.